cypress.json

```diff
@@ -5,5 +5,6 @@
     "pluginsFile": "./tests/cypress/plugins/index.js",
     "screenshotsFolder": "./tests/cypress/screenshots",
     "supportFile": "./tests/cypress/support/index.js",
-    "videoFolder": "./tests/cypress/videos"
+    "videoFolder": "./tests/cypress/videos",
+    "numTestsKeptInMemory": 0
```

Comment:
Some annoying tidbits related to Cypress -- it seems the additional tests tipped it over into Heap Overflow land.